sys-boot/gnu-efi/gnu_efi-3.0.4.recipe

```diff
@@ -0,0 +1,29 @@
+SUMMARY="GNU EFI Library - Kernel support"
+DESCRIPTION="Develop EFI applications for ARM-64, ARM-32, x86_64, IA-64 (IPF), \
+and IA-32 (x86) platforms using the GNU toolchain and the EFI development \
+environment."
+HOMEPAGE="https://sourceforge.net/projects/gnu-efi/"
+COPYRIGHT="1999-2007 Hewlett-Packard Co.
+	2006-2010 Intel Co."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://downloads.sf.net/gnu-efi/gnu-efi-$portVersion.tar.bz2"
+CHECKSUM_SHA256="51a00428c3ccb96db24089ed8394843c4f83cf8f42c6a4dfddb4b7c23f2bf8af"
+SOURCE_DIR="gnu-efi-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
```

Comment:
There is a reason this was called gnuefi_kernel. We have no need for the libraries, etc. for the bootloader, only the headers, linker script, etc. A separate recipe for the libraries should be done instead.